app/src/main/java/org/gdg/frisbee/android/api/GdgXHub.java

```diff
@@ -26,6 +26,13 @@
                                                @Path("start") final DateTime start,
                                                @Path("end") final DateTime end);
 
+    @GET("chapters/{chapterId}/events")
+    Call<PagedList<Event>> getChapterAllEventList(@Path("chapterId") final String chapterId,
```

Comment:
Yeah, can do that. Just thought it would be good to have a more generic method from start.